style.css

```diff
@@ -1,413 +1,889 @@
-  margin-top: 15px;
+.desk-li-menu .desk-li {
+  text-align: left;
+  list-style: none;
+  padding-right: 25px;
+  font-family: "Inter", sans-serif;
+  font-weight: 500;
+  font-size: 32px;
 }
 
-@media screen and (min-width: 1024px) {
-  .header {
-    padding: 10px 12px;
-    margin: 15px 154px;
+@media (min-width: 990px) {
+  .mob-img {
+    display: none;
+  }
+
+  .close {
+    display: none;
+  }
+
+  .desk-li {
+    display: flex;
+    justify-content: start;
+    list-style: none;
+    gap: 15px;
+  }
+
+  .desk-head {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
     display: flex;
     align-items: center;
+    text-align: center;
+    color: #fff;
+    margin-right: 54px;
+  }
+
+  .name-me {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 20px;
+    line-height: 20px;
+    text-align: center;
+    padding: 12px;
+    margin-left: 120px;
   }
 
-  .desktop-navigation > ul {
+  .desk-img {
+    color: #67798e;
+    width: 23px;
+    height: inherit;
+  }
+}
+
+@media (max-width: 990px) {
+  .desk-head {
+    display: none;
+  }
+
+  .work-on {
+    align-self: center;
+  }
+}
+
+@media (min-width: 990px) {
+  .heading {
     display: flex;
-    transform: translateY(-50%);
-    font-family: "Poppins", sans-serif;
+    flex-direction: row-reverse;
+    justify-content: start;
+    gap: 89px;
+    background-image: url(icons/Illustration-header@2x.png);
+    background-repeat: no-repeat;
+    background-position-y: -668px;
+    background-position-x: 510px;
+  }
+
+  .main-text {
+    display: flex;
+    flex-direction: column;
+    width: 485px;
+    margin-top: 65px;
+  }
+
+  .text {
+    font-family: "Crete Round", sans-serif;
+    font-size: 72px;
+    line-height: 81.5px;
+    line-height: 89%;
+    text-align: left;
+    vertical-align: top;
+    margin-bottom: 20px;
+  }
+
+  .text1 {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
     font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+    display: flex;
+    align-items: center;
+    color: #fff;
+    margin-bottom: 20px;
+  }
+
+  .text2 {
+    font-family: "Inter", sans-serif;
     font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+  }
+
+  .contacts {
+    display: flex;
+    flex-direction: column;
+    margin-top: 140px;
+  }
+}
+
+@media (min-width: 990px) {
+  .work-section {
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+    padding: 100px 100px;
+    background-image: none;
+  }
+
+  .MY-works {
+    display: flex;
+    gap: 52px;
+    margin-bottom: 30px;
+  }
+
+  .part1 {
+    display: flex;
+    align-items: center;
+    gap: 20px;
+  }
+
+  h2 {
+    height: 55px;
+  }
+
+  .recent-work {
+    font-family: "Crete Round", sans-serif;
+    font-size: 40px;
+    text-align: left;
+    color: #091e42;
+  }
```

Comment:
- [ ] Because this is your last chance of requesting a review. I will not require another change, so I will make it optional:
- You haven't implemented the same fonts that are provided in the Figma design. I would highly recommend changing the fonts before merging the pull request:

<img width="796" alt="image" src="https://user-images.githubusercontent.com/77190203/155486714-9152587f-2c1e-4cda-ae8c-e69e8ffbfd49.png">